src/components/timetable/timetableContainer.js

```diff
@@ -34,20 +34,38 @@ function groupDepartures(departures) {
 }
 
 function getNotes(isSummerTimetable) {
+    // because apparently there may be many
+    let didForceShowPNote = false;
```

Comment:
p-notes, or notes about friday departues, are sometimes entered into JORE with a type that precludes them from being visible for the public. The noteType needs to contain the letter Y for the note to be shown.
There may also be multiple notes of the same kind, but they are only shown under one column. Thus I need to track if a note has already been force-shown.